### DISCUSSION TITLE
HTTP2 behind NGINX cache/reverse proxy

### DISCUSSION THREAD

**Discussion by naanlizard** [OvenMediaLabs/OvenMediaEngine#1695] [Help]
I've got OME set up with a simple origin and edge combined instance behind an nginx reverse proxy subdomain

I've been taking a closer look at logging/etc lately and found that I'm getting HTTP1.1 requests for llhls chunks with my current config.

Here's my [Server.xml](https://gist.github.com/naanlizard/e028a2302eb10004b4bfa41055800d67) for reference. The big question here is, do I need 

```
        <HTTP2>
            <Enable>true</Enable>
        </HTTP2>
```

somewhere, as the [docs suggest](https://airensoft.gitbook.io/ovenmediaengine/configuration#configuration-example)? 

Or is it enabled by default, like LLHLS must be ? 

I understand it's only available on the TLS port, correct?

Does any of this even matter? e.g. if the nginx->OME connection is http and only http 1.1, is that going to be noticeable compared to http2?

**Comment by getroot** [comment]
An HTTP server will only know if a client wants to connect over http/2 if it uses TLS. There will be no noticeable performance improvement between servers, since clients will be using multiple connections on top of http1.1 anyway.
